test/unit-tests/cms-driver/cms-driver.test.ts

```diff
@@ -62,19 +62,24 @@ suite('CMake-Server-Driver tests', () => {
 
   setup(async function(this: Mocha.IBeforeAndAfterContext, done) {
     driver = null;
-
-    if (!cleanupBuildDir(path.join(defaultWorkspaceFolder, 'build'))) {
+    let isDone = false;
+    if (!isDone && !cleanupBuildDir(path.join(defaultWorkspaceFolder, 'build'))) {
       done('Default build folder still exists');
+      isDone = true;
     }
 
-    if (!cleanupBuildDir(path.join(emptyWorkspaceFolder, 'build'))) {
+    if (!isDone && !cleanupBuildDir(path.join(emptyWorkspaceFolder, 'build'))) {
```

Comment:
I don't understand the logic change here.  Shouldn't we attempt to clean up all build folders?